plugins/auth0/pkg/auth0client/auth0client.go

```diff
@@ -0,0 +1,33 @@
+package auth0client
+
+import (
+	"context"
+
+	"github.com/auth0/go-auth0/management"
+)
+
+type Auth0Client struct {
```

Comment:
Why do we need this type? Why not return `*management.Management` from `New()`?

Reply:
was talking with @BogdanIrimie about having a common interface for all fetcher clients at some point, so a start would be having a custom client for each plugin